prowler/providers/aws/services/dms/dms_service.py

```diff
@@ -49,6 +51,31 @@ def __describe_replication_instances__(self, regional_client):
                 f"{regional_client.region} -- {error.__class__.__name__}[{error.__traceback__.tb_lineno}]: {error}"
             )
 
+    def __describe_endpoints__(self, regional_client):
+        logger.info("DMS - Describing DMS Endpoints...")
+        try:
+            describe_endpoints_paginator = regional_client.get_paginator(
+                "describe_endpoi4nts"
```

Comment:
```suggestion
                "describe_endpoints"
```
Have you verified that the check works in AWS?

Reply:
Hi, i am testing this on AWS now

Reply:
sorry i realised this. this must be a typo be before i make my first commit. i remember, i spent some time time figuring out how to get_paginate() method works and how ti fetches the endpoint(as im dealing with dicts)